demo-simple/src/main/java/com/novoda/downloadmanager/demo/CustomNotificationConfigFactory.java

```diff
@@ -0,0 +1,50 @@
+package com.novoda.downloadmanager.demo;
+
+import android.app.Notification;
+import android.content.Context;
+import android.support.v4.app.NotificationCompat;
+import android.support.v4.app.NotificationManagerCompat;
+
+import com.novoda.downloadmanager.DownloadBatchStatus;
+import com.novoda.downloadmanager.DownloadBatchTitle;
+import com.novoda.downloadmanager.NotificationConfig;
+import com.novoda.downloadmanager.NotificationCustomizer;
+import com.novoda.notils.logger.simple.Log;
+
+public final class CustomNotificationConfigFactory {
+
+    private static final boolean NOT_INDETERMINATE = false;
+
+    public static NotificationConfig<DownloadBatchStatus> createDownloadNotificationConfig(Context context, String channelId, String channelDescription, final int iconDrawable) {
```

Comment:
If we are planning to have lint etc to run on this later then we will need to include `private` constructors on our factories. Can you add one here please 😃

Reply:
I normally add a comment saying "Uses static factory methods."